From e71c17779387a7acff1cce2bb1cc04b096642891 Mon Sep 17 00:00:00 2001
From: Kent Russell <kent.russell@amd.com>
Date: Tue, 5 Jul 2016 10:38:43 -0400
Subject: [PATCH 1747/2094] Remove lazy param to ttm_bo_wait

This has been removed from the ttm_bo_wait function, so change our call
accordingly.

Change-Id: Ic163b3c0d6c16a6423ed5a89d7864f678f22b7f1
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c | 2 +-
 drivers/gpu/drm/radeon/radeon_kfd.c              | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c
index 8ec4859..2826e50 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c
@@ -526,7 +526,7 @@ static void unreserve_bo_and_vms(struct bo_vm_reservation_context *ctx,
 		int ret;
 
 		list_for_each_entry(entry, &ctx->list, head) {
-			ret = ttm_bo_wait(entry->bo, false, false, false);
+			ret = ttm_bo_wait(entry->bo, false, false);
 			if (ret != 0)
 				pr_err("amdkfd: Failed to wait for PT/PD update (err == %d)\n",
 				       ret);
diff --git a/drivers/gpu/drm/radeon/radeon_kfd.c b/drivers/gpu/drm/radeon/radeon_kfd.c
index c347b39..d557a69 100644
--- a/drivers/gpu/drm/radeon/radeon_kfd.c
+++ b/drivers/gpu/drm/radeon/radeon_kfd.c
@@ -1278,7 +1278,7 @@ static int map_bo_to_gpuvm(struct radeon_device *rdev, struct radeon_bo *bo,
 
 	list_for_each_entry(lobj, &list, tv.head) {
 		struct radeon_bo *bo = lobj->robj;
-		ret = ttm_bo_wait(&bo->tbo, true, false, false);
+		ret = ttm_bo_wait(&bo->tbo, false, false);
 		if (ret != 0) {
 			pr_err("amdkfd: Failed to wait for PT/PD update (err == %d)\n",
 					ret);
-- 
2.7.4

